src/medCore/database/medDatabaseRemover.cpp

```diff
@@ -275,11 +253,16 @@ void medDatabaseRemover::removeStudy ( int patientDbId, int studyDbId )
 
     query.prepare ( "SELECT thumbnail, name, uid FROM " + d->T_STUDY + " WHERE id = :id " );
     query.bindValue ( ":id", studyDbId );
+
     EXEC_QUERY ( query );
 
     if ( query.next() )
     {
-        removeThumbnailIfNeeded(query);
+        medAbstractDbController * dbc = medDataManager::instance()->controllerForDataSource(d->index.dataSourceId());
+        if (dbc->metaData(d->index,  medMetaDataKeys::StudyID.key()).toInt() == studyDbId)
```

Comment:
It's not very clear but when you move a file from a serie to an other serie/patient, `studyDbId` is not equal to `dbc->... `.